test/spec/modules/adbutlerBidAdapter_spec.js

```diff
@@ -1,516 +1,274 @@
-describe('adbutler adapter tests', function () {
-  var expect = require('chai').expect;
-  var adapter = require('modules/adbutlerBidAdapter');
-  var adLoader = require('src/adloader');
-  var bidmanager = require('src/bidmanager');
-
-  describe('creation of bid url', function () {
-    var stubLoadScript;
-
-    beforeEach(function () {
-      stubLoadScript = sinon.stub(adLoader, 'loadScript');
-    });
-
-    afterEach(function () {
-      stubLoadScript.restore();
-    });
-
-    if (typeof ($$PREBID_GLOBAL$$._bidsReceived) === 'undefined') {
-      $$PREBID_GLOBAL$$._bidsReceived = [];
-    }
-    if (typeof ($$PREBID_GLOBAL$$._bidsRequested) === 'undefined') {
-      $$PREBID_GLOBAL$$._bidsRequested = [];
-    }
-    if (typeof ($$PREBID_GLOBAL$$._adsReceived) === 'undefined') {
-      $$PREBID_GLOBAL$$._adsReceived = [];
-    }
-
-    it('should be called', function () {
-      var params = {
-        bidderCode: 'adbutler',
-        bids: [
-          {
-            bidId: '3c9408cdbf2f68',
-            sizes: [[300, 250]],
-            bidder: 'adbutler',
-            params: {
-              accountID: '167283',
-              zoneID: '210093'
-            },
-            requestId: '10b327aa396609',
-            placementCode: '/123456/header-bid-tag-1'
-          }
-
-        ]
-      };
-
-      adapter().callBids(params);
-
-      sinon.assert.called(stubLoadScript);
-    });
-
-    it('should populate the keyword', function() {
-      var params = {
-        bidderCode: 'adbutler',
-        bids: [
-          {
-            bidId: '3c9408cdbf2f68',
-            sizes: [[300, 250]],
-            bidder: 'adbutler',
-            params: {
-              accountID: '167283',
-              zoneID: '210093',
-              keyword: 'fish'
-            },
-            requestId: '10b327aa396609',
-            placementCode: '/123456/header-bid-tag-1'
-          }
-
-        ]
-      };
-
-      adapter().callBids(params);
-
-      var requestURI = stubLoadScript.getCall(0).args[0];
-
-      expect(requestURI).to.have.string(';kw=fish;');
-    });
-
-    it('should use custom domain string', function() {
-      var params = {
-        bidderCode: 'adbutler',
-        bids: [
-          {
-            bidId: '3c9408cdbf2f68',
-            sizes: [[300, 250]],
-            bidder: 'adbutler',
-            params: {
-              accountID: '107878',
-              zoneID: '86133',
-              domain: 'servedbyadbutler.com.dan.test'
-            },
-            requestId: '10b327aa396609',
-            placementCode: '/123456/header-bid-tag-1'
+import {expect} from 'chai';
+import adapterManager from 'src/adaptermanager';
+import bidManager from 'src/bidmanager';
```

Comment:
We probably should have been more explicit about this in our docs, but... the `bidmanager` is going away in 1.0. The good news is that your tests can be simplified.

New your tests can just create objects with dummy data (bid request, server response, etc), pass them into the `spec` methods directly, and then make assertions on the return values. No need to worry about mocks, fake servers, or anything like that. The tests on the `bidderFactory` will make sure that your adapter's bids make it into the auction. 

Check out the [appnexusAst adapter](https://github.com/prebid/Prebid.js/blob/master/test/spec/modules/appnexusAstBidAdapter_spec.js) for some working examples..